.github/workflows/refresh-tiny-kb.yml

```
@@ -0,0 +1,53 @@
+# Don't queue concurrent refreshes — if two PRs merge back-to-back,
+# the latest one wins and the older invocation is cancelled.
+concurrency:
+  group: refresh-tiny-kb
+  cancel-in-progress: true
+
+jobs:
+  notify:
+    runs-on: ubuntu-latest
+    # Skip cleanly when the bot isn't configured yet so this PR can
+    # merge before Tiny is deployed.
+    if: vars.TINY_WEBHOOK_URL != '' || secrets.TINY_WEBHOOK_URL != ''
```

actionlint (1.7.12)

[error] context "secrets" is not allowed here. available contexts are "github", "inputs", "needs", "vars". see https://docs.github.com/en/actions/learn-github-actions/contexts#context-availability for more details

(expression)